File: src/main/java/pvz/community/launcher/configuration/LaunchConfiguration.java
package pvz.community.launcher.configuration;

import pvz.community.logger.GlobalConsole;

import java.io.File;
import java.io.IOException;

public class LaunchConfiguration {

    public File configFolder = new File("pvz-community");
    public File launchConfigurationJSON = new File(configFolder, "launch.json");

    public Configuration launchConfig = Configuration.newConfiguration(launchConfigurationJSON);

    public void saveLaunchConfiguration() throws IOException {
        if (!configFolder.exists()) {
            configFolder.mkdirs();
        }
        GlobalConsole.getLogger().info("Saving launch configuration...");


    }

}

File: src/main/java/pvz/community/launcher/configuration/LaunchConfiguration.kt
package pvz.community.launcher.configuration

import pvz.community.launcher.configuration.Configuration.Companion.newConfiguration
import pvz.community.utils.Logger.info
import java.io.File
import java.io.IOException

class LaunchConfiguration {
    var configFolder = File("pvz-community")
    var launchConfigurationJSON = File(configFolder, "launch.json")
    var launchConfig = newConfiguration(launchConfigurationJSON)

    @Throws(IOException::class)
    fun saveLaunchConfiguration() {
        if (!configFolder.exists()) {
            configFolder.mkdirs()
        }
        info("Saving launch configuration...")
    }
}